BootloaderCommonPkg/Include/Guid/MpCpuTaskInfoHob.h

```diff
@@ -0,0 +1,50 @@
+/** @file
+  This file defines the hob structure for the AP task.
+
+  Copyright (c) 2021, Intel Corporation. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef __MP_CPU_TASK_INFO_H__
+#define __MP_CPU_TASK_INFO_H__
+
+
+///
+/// Boot loader MP CPU task Information GUID
+///
+extern EFI_GUID gLoaderMpCpuTaskInfoGuid;
+
+typedef enum {
+  EnumCpuBusy = 0,
+  EnumCpuReady,
+  EnumCpuStart,
+  EnumCpuEnd,
+} CPU_STATE;
+
+typedef UINT32 (*CPU_TASK_PROC)          (UINT32 Arg);
+
+typedef struct {
+  // Refer CPU_STATE
+  UINT32           State;

```

Comment:
considering we are supporting 64bit.  
Should we to have all argument to update to 64bit instead ?  
It is HOB now, need to avoid changing later   // CPU task function
typedef struct {
  UINT8             State;
  UINT8             Request;
  UINT8             Reserved[6];
  UINT64           CProcedure;
  UINT64           Argument;
  UINT64           Result;
} CPU_TASK;on.

Reply:
yes. we could update to use UINT64. what the Request field is used for?